app/build.gradle

```diff
@@ -36,7 +36,11 @@ dependencies {
 
     implementation 'com.squareup.retrofit2:retrofit:2.7.1'
     implementation 'com.squareup.retrofit2:converter-gson:2.7.1'
+    //location service library
+    implementation'com.google.android.gns:play-services-location:17.0.0'
 
```

Comment:
I added a location library to your build.gradle because If your app needs to access the user's location, you must request permission by adding the relevant Android location permission to your app.